### DISCUSSION TITLE
Can't play the video file

### DISCUSSION THREAD

**Discussion by huahuasousou** [photoprism/photoprism#2775] [General]
#### 1. What is not working as documented?

When the video is imported into photoprims, the time length cannot be displayed, show the 0:00, and video cannot be played, whatever on PC or mobile platforms. But the file to play directly in the system of the same device, it normal.

#### 2. How can we reproduce it?

1. From following down the video:
2. Import to the photoprims
3. Try to play it

#### 3. What behavior do you expect?

The problem can be found and fixed

#### 4. What could be the cause of your problem?

I hope you can check the file and make the player more generalization

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

https://drive.google.com/file/d/1xJt9XIq-N1rXnZp2kFilFlCH_GtvYUqS/view?usp=sharing

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64
[PhotoPrism® CE](http://localhost:2342/about)
Build [220901-f493607b0](https://docs.photoprism.app/release-notes/)

(b) Database Type & Version: MariaDB

(c) Operating System Types & Versions: Linux, Ubuntu20.04LTS

(d) Browser Types & Versions: Chrome on the ipad pro 12.9 inch 2018(IpadOS15.7), iphone13 pro and pro max(Ios16.0.2), and PC platform Ubuntu20.04 and windows10(Chrome)

(e) No Ad Blockers, Browser Plugins, and/or Firewall Software

#### 7. On what kind of device is PhotoPrism installed?

(a) Device / Processor Type: Inter I7 6700HQ, DDR3 16GB RAM, GTX960M GPU

(b) Physical Memory & Swap Space in 16GB & 20GB

(c) Storage Type: HDD in the Network Storage, use OMV6 create a Samba server

(d) Anything else that might be helpful to know?

No

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

No

**Comment by graciousgrey** [comment]
The video can be played without issues on our test instances.

Please note: The first time the video is played it may need to be transcoded and it will take a while until playback starts, see

https://docs.photoprism.app/user-guide/organize/video/#transcoding

**Reply by huahuasousou** [reply]
I didn't want this file to be played after ffmpeg converted, I just wanted to play it directly.

After my week of testing I found out why. The reason for the video not playing has to do with the bitrate. When the bitrate was 150mbps everything was fine and it could be played directly. When the bitrate grows to 200mbps it will not play.

I'm very intersting and would like to know if this is a limitation of ios or photoprism. I have tried setting the threshold for "ffmpeg conversion" to 500mbps to make sure it doesn't trigger.
because my video are HLG or HDR10 format, and some file are 60fps, so I dont want to convert it to a lower format (h264 8bit 30fps).

**Reply by lastzero** [reply]
It's not a limitation, but a feature to make sure you can actually play the video using a commercially available internet connection in case you happen not to sit right in front of the server with a 1G connection and a client device who can handle the bitrate / framerate. If you are sure your internet connection and device will always be fast enough (we do not somehow automatically detect it right now), then just keep the high settings for PHOTOPRISM_FFMPEG_BITRATE and you should be fine. When in doubt, check the source code and related issues comments as we have discussed this with our community before.

**Reply by huahuasousou** [reply]
I confirmed that I don't have enable the ffmpeg convert, and that PHOTOPRISM_FFMPEG_BITRATE is set to 500. This doesn't trigger a convert. and I will working in the LAN so I dont need convert function. but I can play the 200mbps hevc in any ios/ipados device. could you check it again?

**Comment by abugray** [comment]
> I confirmed that I don't have enable the ffmpeg convert, and that PHOTOPRISM_FFMPEG_BITRATE is set to 500. This doesn't trigger a convert. and I will working in the LAN so I dont need convert function. but I can play the 200mbps hevc in any ios/ipados device. could you check it again?

How are you resolve this? I have same problem with hdr videos.

**Reply by graciousgrey** [reply]
The problem mentioned here has been solved with https://github.com/photoprism/photoprism/issues/3008. 

FFmpeg must be enabled in any case. However not all videos need to be transcoded in order to play if they are supported by your browser and do not exceed the configured bitrate limit. See also:

https://docs.photoprism.app/user-guide/organize/video/#transcoding
https://docs.photoprism.app/getting-started/troubleshooting/#videos-dont-play
